Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -211,6 +226,10 @@ void MemoryWidget::ConnectWidgets()
   connect(m_find_next, &QPushButton::pressed, this, &MemoryWidget::OnFindNextValue);
   connect(m_find_previous, &QPushButton::pressed, this, &MemoryWidget::OnFindPreviousValue);
 
+  for (auto* radio :
+       {m_address_space_effective, m_address_space_auxiliary, m_address_space_physical})
+    connect(radio, &QRadioButton::toggled, this, &MemoryWidget::OnAddressSpaceChanged);
```

Comment:
The body should be braced here, since the range expression travels more than one line.